cmd/aigw/ai-gateway-default-resources.yaml

```diff
@@ -90,6 +90,49 @@ spec:
           socket_address:
             address: 127.0.0.1
             port_value: 9901
+  telemetry:
+    accessLog:
+      settings:
+        - sinks:
+            - type: File
+              file:
+                path: /dev/stdout
+          format:
+            type: JSON
+            json:
+              # AI specific fields. The properties in the dynamic metadata expressions must match the ones
+              # defined in the AIGatewayRoute llmRequestCosts field.
+              genai_model_name: "%REQ(X-AI-EG-MODEL)%"
+              genai_model_name_override: "%DYNAMIC_METADATA(io.envoy.ai_gateway:model_name_override)%"
+              genai_backend_name: "%DYNAMIC_METADATA(io.envoy.ai_gateway:backend_name)%"
+              genai_tokens_total: "%DYNAMIC_METADATA(io.envoy.ai_gateway:llm_total_token)%"
+              genai_tokens_input: "%DYNAMIC_METADATA(io.envoy.ai_gateway:llm_input_token)%"
+              genai_tokens_output: "%DYNAMIC_METADATA(io.envoy.ai_gateway:llm_output_token)%"
+              # Default fields
+              start_time: "%START_TIME%"
+              method: "%REQ(:METHOD)%"
+              x-envoy-origin-path: "%REQ(X-ENVOY-ORIGINAL-PATH?:PATH)%"
+              protocol: "%PROTOCOL%"
+              response_code: "%RESPONSE_CODE%"
+              response_flags: "%RESPONSE_FLAGS%"
+              response_code_details: "%RESPONSE_CODE_DETAILS%"
+              connection_termination_details: "%CONNECTION_TERMINATION_DETAILS%"
+              upstream_transport_failure_reason: "%UPSTREAM_TRANSPORT_FAILURE_REASON%"
+              bytes_received: "%BYTES_RECEIVED%"
+              bytes_sent: "%BYTES_SENT%"
+              duration: "%DURATION%"
+              x-envoy-upstream-service-time: "%RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)%"
```

Comment:
also good to add the `x-request-tx-duration-ms` this is the preprocessing time before hitting the upstream

Reply:
I think the preprocessing time is critical at least for us, user want to know how much time is spent on the ai gateway extproc vs upstream. cc @johnugeorge @gavrissh

Reply:
That's fair, but this is a different issue. We're now talking about EAIGW needing a different set of defaults for access logs, because the current ones lack important info.

I think it's better to keep the scope of this PR to explain how to populate the filter metadata info in the access logs, and discuss separately what the default access logs config should be for EAIGW.